docs/source/release/v6.11.0/Workbench/New_features/37519.rst

```diff
@@ -0,0 +1 @@
+- Updated compiler on Linux to gcc version 12, which should improve performance in some circumstances
```

Comment:
Should this be in `Framework/Dependancies`?

Reply:
Release note moved